src/components/Header/components/NavButtonsRightCorner/navButtons.js

```diff
@@ -0,0 +1,324 @@
+import React from 'react';
+import ListItem from '@material-ui/core/ListItem';
+import ListItemText from '@material-ui/core/ListItemText';
+import { NavLink } from 'react-router-dom';
+import user from '../../../../services/user';
+
+/**
+ * Creates the Links button.
+ *
+ * Depending on the status of the network, authentication, and user role, the Links button is created.
+ *
+ * @param {String} networkStatus The status of the network. Either 'online' or 'offline'.
+ * @param {Function} onLinkClickOne Determines what occurs when the user clicks on this button.
+ * @param {Function} onLinkClickTwo Determines what occurs when the user clicks on this button
+ *                                  after onLinkClickOne occurs.
+ * @param {Function} openDialogBox Sets the data of a dialog box to be shown if the user clicks on
+ *                                this button.
+ *
+ * @return {JSX} The JSX of the Links button.
+ */
+export function createLinksButton(networkStatus, onLinkClickOne, onLinkClickTwo, openDialogBox) {
+  let linksButton =
+    // Network Status: Online
+    networkStatus === 'online' ? (
+      <ListItem
+        divider
+        button
+        onClick={() => {
+          onLinkClickOne();
+          onLinkClickTwo();
+        }}
+      >
+        <ListItemText primary="Links" />
+      </ListItem>
+    ) : (
+      // Network Status: Offline
+      <div
+        onClick={() => {
```

Comment:
I would display the Links Dialog Box while offline but there are no links that lead to a page on 360. They're all external links that would fail to load because the user is offline. That's why I disabled the Links Dialog Box completely when offline